cpmpy/solvers/gurobi.py

```diff
@@ -281,6 +282,9 @@ def _post_constraint(self, cpm_expr):
             Solvers can raise 'NotImplementedError' for any constraint not supported after transformation
         """
         from gurobipy import GRB
+        # True or False
+        if is_bool(cpm_expr):
+            return self.grb_model.addConstr(cpm_expr)
```

Comment:
Maybe better use the expr.is_bool() method, to be consistent with other similar checks that are done.